lib/modules/nightMode.js

```diff
@@ -121,17 +127,20 @@ module.beforeLoad = () => {
 		removeNightMode();
 	}
 
-	toggle = new CustomToggles.Toggle('nightMode', i18n('nightModeToggleText'), module.options.nightModeOn.value);
-	toggle.onToggle(type => {
-		if (type === 'manual') overrideNightMode();
-		Options.set(module, 'nightModeOn', toggle.enabled);
-	});
-	toggle.onStateChange(() => {
-		if (toggle.enabled) addNightMode();
-		else removeNightMode();
-		StyleTweaks.toggleSubredditStyleIfNecessary();
-	});
-	toggle.addCLI('ns');
+	// Add the night mode toggle in the gear menu *only for the old design*
+	if (className() === 'res-nightmode') {
```

Comment:
Use `isAppType('d2x')` (desktop2x, the codename of the redesign...maybe this could be more intuitive) or `isAppType('r2')` instead of comparing the classname

Reply:
Fixed. It does look cleaner than using a triple-equality comparison

Reply:
H